www/kudp02/ukol-3.css

```diff
@@ -0,0 +1,126 @@
+p, div {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+#text-2 {
+color:red;
+}
+
+#text-3 {
+    color: (0,255,0);
+}
+
+#text-4 {
+    color:#0000ff;
+    font-weight:bold;
+}
+
+#text-5 {
+    font-size:18px;
+}
+
+#text-6 {
+    font-size:24px;
+    font-style: italic;
+}
+
+#text-7 {
+    color:white;
+    background-color: brown;
+}
+
+#text-8 {
+    text-align: center;
+    line-height: 32px;
+}
+
+#text-9 {
+    text-align: justify;
+    text-transform: uppercase;
+}
+
+#box-1 {
+    width: 400px;
+    height: 300px;
+    border-style: solid;
+    border-width: 1px;
+    border-color: purple;
+    padding: 30px;
+}
+
+#box-2 {
+    width: 500px;
+    height: 200px;
+    background-color: red;
+    margin: 100px;
+}
+
+#box-3 { 
+    visibility: hidden;
+}
+
+#box-4 {
+    display: block;
+    width: 100%;
```

Comment:
chybi box-sizing: border-box, proto se stalo, ze sirka tohohle boxu je vetsi nez sirka jeho containeru, i kdyz to ma byt 100% , protoze je tam navic padding a border...

box-sizing: border-box je dobre davat do wildcard selektoru (hvezdicka)